website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -14,6 +14,57 @@ You can create a Kubeflow Pipeline built entirely using SageMaker components, or
 
 There are two versions of SageMaker components - boto3 based v1 components and SageMaker Operator for K8s (ACK) based v2 components. 
 
+
+## Configure Permissions
+
+Configure RBAC permissions for the service account used by kubeflow pipeline pods in the user/profile namespace. The pipeline runs are executed in user namespaces using the default-editor Kubernetes service account.
+
+
+Set the environment variable value for PROFILE_NAMESPACE(e.g. kubeflow-user-example-com) according to your profile and SERVICE_ACCOUNT name according to your installation:
```

Comment:
@ryansteakley this is not needed for for full Kubeflow, these permissions are preconfigured. 

Create a section for _Configuration for SageMaker components version 2_ and just mention there is no extra configuration needed for using components version 2 if you have installed the AWS distribution of Kubeflow.